project.clj

```diff
@@ -25,53 +23,13 @@
                  [org.webjars/SlickGrid "2.1"]
                  ;; CLIENT REQUIREMENTS
                  [cljs-http "0.1.17"]]
-  :plugins [[lein-cljsbuild "1.0.5"]
-            [com.keminglabs/cljx "0.6.0" :exclusions [org.clojure/clojure]]]
-  :source-paths ["src/cljs"
-                 "target/generated/src/clj"
-                 "target/generated/src/cljs"
-                 "target/classes"]
+  :source-paths ["src"]
+  :plugins [[lein-cljsbuild "1.0.5"]]
   :clean-targets ["out/hatti" "out/hatti.js"]
-  :prep-tasks [["cljx" "once"] "javac" "compile"]
-  :cljx {:builds [{:source-paths ["src/cljx"]
-                   :output-path "target/generated/src/clj"
-                   :rules :clj}
-                  {:source-paths ["src/cljx"]
-                   :output-path "target/generated/src/cljs"
-                   :rules :cljs}]}
-  :cljsbuild {
-    :builds [{:id "stolen"
-              :source-paths ["src/cljs"
```

Comment:
I got confirmed from @prabhasp He moved them to gh-pages already but hasn't cleaned up. This is the clean up